draft-packages/table/KaizenDraft/Table/styles.scss

```diff
@@ -80,20 +68,13 @@ $row-height: 60px;
 .card {
   @include button-reset;
   background: $kz-color-white;
-  border: solid 1px $ca-border-color;
-  // Reverting back from $kz-shadow-small-box-shadow as an interim.
-  // @see https://cultureamp.slack.com/archives/CETLTFG9G/p1588658249280300?thread_ts=1588587636.273700&cid=CETLTFG9G
-  box-shadow: $ca-box-shadow;
+  border: solid 1px rgba($kz-color-wisteria-700, 0.1);
   transition: box-shadow $ca-duration-rapid, border-color $ca-duration-rapid,
     margin $ca-duration-rapid, padding $ca-duration-rapid,
     width $ca-duration-rapid;
   // These css properties are required for when the rows are anchor elements
   composes: anchorReset;
-
-  &:first-child {
-    border-top-left-radius: $kz-border-solid-border-radius;
-    border-top-right-radius: $kz-border-solid-border-radius;
-  }
+  box-shadow: 0px 4px 4px $box-shadow-color-sm;
```

Comment:
I think moving this line so that it's not thought of as combined with lines 75 + 76 would be good

Reply:
Good idea.